packages/base/src/dialogs/symbology/components/color_ramp/ColorRampValueControls.tsx

```diff
@@ -0,0 +1,79 @@
+import { Button } from '@jupyterlab/ui-components';
+import React from 'react';
+
+import { IColorRampValueControlsProps } from '@/src/types';
+
+export const ColorRampValueControls: React.FC<IColorRampValueControlsProps> = ({
+  min,
+  setMin,
+  max,
+  setMax,
+  rampDef,
+  initialMin,
+  initialMax,
+  renderType,
+}) => {
+  return (
+    <>
+      <div className="jp-gis-symbology-row">
+        <label htmlFor="min-value">Min Value:</label>
+        <input
+          id="min-value"
+          type="number"
+          value={min ?? ''}
+          onChange={e =>
+            setMin(
+              e.target.value !== '' ? parseFloat(e.target.value) : undefined,
+            )
+          }
+          className="jp-mod-styled"
+          placeholder="Enter min value"
+        />
+      </div>
+
+      {rampDef.type === 'Divergent' && renderType === 'Graduated' && (
+        <div className="jp-gis-symbology-row">
+          <label htmlFor="critical-value">Critical Value:</label>
+          <input
```

Comment:
Maybe this shouldn't be an input; it's just for display and that should be more clear to the user.